src/core/config/styles.js

```diff
@@ -27,6 +42,8 @@ function getCurrentThemeColor(isContrastMode) {
 
 export function getCurrentTheme(isContrastMode) {
```

Comment:
You can export one theme with both light and dark variants, so you don't need to choose the mode when calling the function. See https://material-ui.com/customization/themes/#type-light-dark-theme

Reply:
It will be done in future

Reply:
Done